go.mod

```diff
@@ -55,6 +57,8 @@ require (
 replace (
 	github.com/piprate/json-gold => github.com/trustbloc/json-gold v0.3.1-0.20200414173446-30d742ee949e
 	golang.org/x/sys => golang.org/x/sys v0.0.0-20200826173525-f9321e4c35a6
+	github.com/kilic/bls12-381 => github.com/trustbloc/bls12-381 v0.0.0-20201008080608-ba2e87ef05ef
+	github.com/phoreproject/bls => github.com/trustbloc/bls v0.0.0-20201008085849-81064514c3cc
```

Comment:
We need an issue to remove this dependency.

Reply:
https://github.com/hyperledger/aries-framework-go/issues/2248